lib/notification-issue.coffee

```diff
@@ -174,12 +174,13 @@ class NotificationIssue
     return packageName if packageName
 
   getPackageName: ->
+    message = @notification.getMessage()
     options = @notification.getOptions()
-    return unless options.stack? or options.detail?
+    return unless message? or options.stack? or options.detail?
 
     packagePaths = @getPackagePathsByPackageName()
     for packageName, packagePath of packagePaths
-      if packagePath.indexOf('.atom/dev/packages') > -1 or packagePath.indexOf('.atom/packages') > -1
+      if packagePath.indexOf(path.join('.atom', 'dev', 'packages')) > -1 or packagePath.indexOf(path.join('.atom', 'packages')) > -1
```

Comment:
The goal of this line is to check if the package is bundled or not right?

There is `atom.packages.isBundledPackage` and `atom.packages.isBundledPackagePath` that might be useful to use instead.